android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java

```diff
@@ -659,6 +682,41 @@ private void registerForCallInvites() {
         Voice.register(accessToken, Voice.RegistrationChannel.FCM, fcmToken, registrationListener);
     }
 
+     /*
+     * Unregister your android device with Twilio
+     *
+     */
+
+    @ReactMethod  //
+    public void unregister(Promise promise) {
+        unregisterForCallInvites();
+        WritableMap params = Arguments.createMap();
```

Comment:
I don't think we need a return map. Most probably for the JS caller it is sufficient to know that the operation succeeded, so `promise.resolve(true)`

Reply:
done